projects/angular-auth-oidc-client/src/lib/config/openid-configuration.ts

```diff
@@ -18,6 +18,13 @@ export interface OpenIdConfiguration {
   authWellknownEndpointUrl?: string;
   authWellknownEndpoints?: AuthWellKnownEndpoints;
 
+    /** 
+     * Override the default Security Token Service wellknown endpoint postfix. 
+     * 
+     * @default /.well-known/openid-configuration
+     */
+  authWellknownUrlSuffix?: string;
```

Comment:
Can you also include this in the docs please? https://github.com/damienbod/angular-auth-oidc-client/blob/main/docs/site/angular-auth-oidc-client/docs/documentation/configuration.md

Reply:
Done.